gap/tools/display.gi

```diff
@@ -727,7 +727,7 @@ function(S, opts)
 
   # TODO make PartialOrderOfDClasses return a digraph
   rel := OutNeighbours(DigraphReflexiveTransitiveReduction(
-                       Digraph(PartialOrderOfDClasses(S))));
+                       DigraphNC(PartialOrderOfDClasses(S))));
   for i in [1 .. Length(rel)] do
```

Comment:
Why did you make this change? I think it would be better not to introduce this change. It gives an additional check that the output of `PartialOrderOfDClasses` is correct.